pkgs/servers/wasmserve/default.nix

```diff
@@ -0,0 +1,22 @@
+{ buildGoModule
+, lib
+, fetchFromGitHub
+}:
+buildGoModule {
+  pname = "wasmserve";
+  version = "2022-11-29";
+  src = fetchFromGitHub {
+    owner = "hajimehoshi";
+    repo = "wasmserve";
+    rev = "689d7ac772c6be55a8deb81885898db78ba81651";
+    sha256 = "sha256-t8QLquAedj57DzE1/8NthxM+r4Hd2GdYTP/sO0XrxHw=";
+  };
+  vendorHash = null;
+  meta = with lib; {
+    description = "An HTTP server for testing Wasm";
+    homepage = "https://github.com/hajimehoshi/wasmserve";
+    license = licenses.asl20;
+    maintainers = [ maintainers.kirillrdy ];
+    platforms = platforms.all;
```

Comment:
This should have been set by buildGoModule by default.

Reply:
removed